yoti_python_sdk/activity_details.py

```diff
@@ -87,17 +110,23 @@ def try_convert_structured_postal_address_to_dict(field):
 
     def ensure_postal_address(self):
         # setting in 'user_profile'  - will be removed once user_profile is removed
-        if config.ATTRIBUTE_POSTAL_ADDRESS not in self.user_profile and config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS in self.user_profile:
-            if config.KEY_FORMATTED_ADDRESS in self.user_profile[config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS]:
-                self.user_profile[config.ATTRIBUTE_POSTAL_ADDRESS] = \
-                    self.user_profile[config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS][
-                        config.KEY_FORMATTED_ADDRESS]
+        if (
+            config.ATTRIBUTE_POSTAL_ADDRESS not in self.user_profile
+            and config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS in self.user_profile
+        ):
+            if (
+                config.KEY_FORMATTED_ADDRESS
+                in self.user_profile[config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS]
+            ):
+                self.user_profile[config.ATTRIBUTE_POSTAL_ADDRESS] = self.user_profile[
+                    config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS
+                ][config.KEY_FORMATTED_ADDRESS]
 
     def __iter__(self):
-        yield 'user_id', self.user_id
-        yield 'parent_remember_me_id', self.parent_remember_me_id
-        yield 'outcome', self.outcome
-        yield 'receipt_id', self.receipt_id
-        yield 'user_profile', self.user_profile
-        yield 'profile', self.profile
-        yield 'base64_selfie_uri', self.base64_selfie_uri
+        yield "user_id", self.user_id
```

Comment:
Should `remember_me_id` be added in here?

Reply:
Yes. Yes it should. Ooops.